CHANGELOG.md

```diff
@@ -117,6 +117,7 @@ Make sure to upgrade to v0.25.0 so that your node will continue to connect once
 - Node will send more peers before disconnecting due to a full peer list
 - Refactor CSRF to use HMAC tokens.
 - Add transaction verification parameters to the `GET /health` response
+- Switch `skycoin-cli` from `urfave/cli` to `spf13/cobra`
```

Comment:
Changelog entry needs to be moved to the unreleased section, and add an entry clarifying that all options must use `--` instead of `-`. Make sure all docs that reference single dash usage are updated

Reply:
`-` is allowed in shorthand